content/docs/guides/adopting/import.md

```diff
@@ -18,204 +20,399 @@ When working with existing resources, there are two primary scenarios:
 * You need to reference existing resources to use as inputs to new resources in Pulumi
 * You need to adopt existing resources under management so they can be managed by Pulumi
 
-For the first situation, consult [the user guide index]({{< relref "/docs/guides/adopting#coexistence" >}}). For the second, let's now see how to adopt existing resources.
+For the first situation, consult [the user guide index]({{< relref "/docs/guides/adopting#coexistence" >}}). For the
+second, let's now see how to adopt existing resources.
 
 ## Adopting Existing Resources
 
-To adopt existing resources so that Pulumi is able to manage subsequent updates to them, Pulumi offers the [`import`]({{< relref "/docs/intro/concepts/programming-model#import" >}}) resource option. This option request that a resource defined in your Pulumi program adopts an existing resource in the cloud provider instead of creating a new one as would normally occur. In keeping with its focus on infrastructure as _code_, Pulumi lets you specify this `import` behavior inside the Pulumi code for your infrastructure deployment, instead of outside of it in a manual workflow.
+To adopt existing resources so that Pulumi is able to manage subsequent updates to them, Pulumi offers the
+[`import`]({{< relref "/docs/reference/cli/pulumi_import" >}}) cli command option. This option request that a resource
```

Comment:
Do we want to _replace_ the resource-level import documentation? I thought we'd want to offer this _in addition_, as an alternative, as the other programmatic model still offers benefits around automating the imports, no?

@lukehoban, curious for your feedback aslo.

Reply:
Yes - I would like to keep both. The resource level approach is still useful for many scenarios, and thus is still the definitive documentation for that capability. 

I would lead with `pulumi import` as that is a simpler starting point.